python/src/FunctionalChaosResult_doc.i.in

```diff
@@ -1,11 +1,82 @@
 %feature("docstring") OT::FunctionalChaosResult
 "Functional chaos result.
 
+Returned by functional chaos algorithms, see :class:`~openturns.FunctionalChaosAlgorithm`.
+
+Available constructors:
+    FunctionalChaosResult(*model, distribution, transformation, inverseTransformation, composedModel, orthogonalBasis, indices, alpha_k, Psi_k, residuals, relativeErrors*)
```

Comment:
no need to list ctors when there is only one

Reply:
Ok. Fixed.